test/CloneFactory.test.js

```diff
@@ -0,0 +1,46 @@
+const SimpleToken = artifacts.require('SimpleToken');
+const SimpleCloneFactory = artifacts.require('SimpleCloneFactory');
+
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('CloneFactory', function ([_, owner]) {
+  let factory;
+
+  describe('factory initialized', function () {
+    beforeEach(async function () {
+      factory = await initFactory();
+    });
+
+    it('should create proxy contract with valid bytecode', async function () {
+      const proxy = await factory.createNew();
+      (await readContractCode(proxy.address)).should.be.equal(expectedBytcode());
+    });
+  });
+
+  const initFactory = async function () {
```

Comment:
I wouldn't abstract this (along with`readContractCode` and `expectedBytecode`) into standalone functions until we actually need them (e.g. to prevent repeated code): I'd rather the tests be as explicit and simple as possible, and this obscures them somewhat.